### PR TITLE
PP3 Appendix

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: piety-promoted-v3/modernized/17-appendix.adoc [hunk 1]
```diff
@@ -220,7 +228,7 @@ in the first year of her travels in England,
 is particularly thought worthy of preservation.
 She seems, previously,
 to have been tried by means of much deprivation of that heavenly comfort and support,
-which she loved and sought.`'
+which she loved and sought.
 
 As I quietly rode along, '`says she,
```
Comment (maijapapaya) [inline piety-promoted-v3/modernized/17-appendix.adoc]: I wasn't sure if this should be embedded with the "says she" part in it.

File: piety-promoted-v3/modernized/17-appendix.adoc [hunk 2]
```diff
@@ -274,6 +282,9 @@ JONAS BINNS, a boy in his fifteenth year, son of Jonathan Binns, of Crawshawboot
 in Lancashire, departed this life the 23rd of the Ninth month, 1760.
 His surviving father gave a testimony concerning him, which I nearly copy.
 
+[.embedded-content-document.testimony]
```
Comment (maijapapaya) [inline piety-promoted-v3/modernized/17-appendix.adoc]: Not sure if this should have been embedded either.

File: piety-promoted-v3/modernized/17-appendix.adoc (hunk 1 above)
Comment (jaredh159) [reply piety-promoted-v3/modernized/17-appendix.adoc]: I think it's fine how you left it. 👍

File: piety-promoted-v3/modernized/17-appendix.adoc (hunk 2 above)
Comment (jaredh159) [reply piety-promoted-v3/modernized/17-appendix.adoc]: Yeah, I think this one could go either way, but I prefer it embedded like you did it 👍

**Pull request merged**